### ISSUE TITLE
Doesn't include ~/.local/bin in PATH when launched with a `toolbox`

### ISSUE THREAD

**Issue by VarLad** [nushell/nushell#5655]
### Describe the bug

Nushell doesn't have ~/.local/bin in PATH when launched with `toolbox enter`. 
I'm aware that this might not be a nushell problem, but was wondering if it was solvable from nushell. 
Other ways I tried launching it:
`toolbox run bash -c "/path/to/nu"`
`toolbox run /path/to/nu`

The above ways work for bash
Here's the output of `$env.PATH`

│ 0 │ /usr/local/sbin │
│ 1 │ /usr/local/bin  │
│ 2 │ /usr/sbin       │
│ 3 │ /usr/bin        │
│ 4 │ /sbin           │
│ 5 │ /bin            │

### How to reproduce

1. Launch nushell with a toolbox using `SHELL=path/to/nushell toolbox enter`
2. Execute `$env.PATH`

### Expected behavior

.local/bin are included in the PATH when launched this way

### Configuration

| key                | value                                           |
| ------------------ | ----------------------------------------------- |
| version            | 0.63.0                                          |
| branch             |                                                 |
| commit_hash        | 06cf3fa5ad8c8b1d1351c81ccea59a1f8d588dc9        |
| build_os           | linux-x86_64                                    |
| build_target       | x86_64-unknown-linux-gnu                        |
| rust_version       | rustc 1.61.0 (fe5b13d68 2022-05-18)             |
| rust_channel       | stable-x86_64-unknown-linux-gnu                 |
| cargo_version      | cargo 1.61.0 (a028ae4 2022-04-29)               |
| pkg_version        | 0.63.0                                          |
| build_time         | 2022-05-25 01:07:50 +00:00                      |
| build_rust_channel | release                                         |
| features           | database, dataframe, default, trash, which, zip |
| installed_plugins  |                                                 |

**Comment by rgwood** [comment]
What is `toolbox`? Could you provide a link?

**Comment by VarLad** [comment]
@rgwood https://github.com/containers/toolbox/ https://containertoolbx.org/
Its the primary mode of working for Fedora SilverBlue (and other immutable distro) users. It can also be used normally by usual distros as well.

**Comment by fdncred** [comment]
if you create a config.nu and env.nu file that has the path items you want, you can probably launch nushell like `toolbox run /path/to/nu --config /path/to/config.nu --env-config /path/to/env.nu`. I think that will work.

**Comment by VarLad** [comment]
@fdncred While that doesn't work, the workaround is to append the `let-env PATH = ($env.PATH | append "some/other/path")` to your `config.nu` file by editing it manually. 
This is inconvenient since I often have to remove the files with every install of nushell in case of a problem (especially with the current releases)

**Comment by VarLad** [comment]
I think its got something to do with the fact of how `nushell` handles containers

**Comment by fdncred** [comment]
`--config` and `--env-config` are how we load pre-configured configuration files as detailed here https://github.com/nushell/nushell/pull/5618. If that's not work, then it's a bug.

**Comment by VarLad** [comment]
@fdncred It does work, but the default `config.nu` doesn't define a hardcoded path to ~/.local/bin
It works if you hardcode the PATH in config.nu but thats really inconvenient since its often needed to be deleted

**Comment by fdncred** [comment]
Nushell is cross platform so hard coding a *nix path for Windows. We do not hard code any paths in the default config file.

**Comment by VarLad** [comment]
@fdncred Yeah. Hence my issue being, nushell can't find the `~/.local/bin` file in containers

**Comment by fdncred** [comment]
Yes, I understand. 

Theoretically we could use conditional compilation to include different config files per platform. I'm just not sure we'd do that.
